### PR TITLE
Hey, look, it's an installfest!

### PR DESCRIPTION
**Pull request by jnf**
@kariabancroft @sudocrystal review pls! ^_^

### CODE REVIEW

File: lessons/06-environment-setup/README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+- To understand the role of and then setup a development environment
+- To understand the role of (and then use!) a _package manager_
+- To understand the function of _ruby_ vs _irb_
+- To understand the purpose of a text editor, in this case _atom_
 
 ## Resources
-1.
+1. [What is _Ruby_? What is _irb_?](notes/what-even-is-ruby.md)
+1. [What is a _text editor_?](notes/what-is-a-text-editor.md)
+1. [Installfest!](notes/installfest.md)
```
Comment (sudocrystal) [inline lessons/06-environment-setup/README.md]: Can we make this follow the other's naming scheme with "Notes: " in front?

File: lessons/06-environment-setup/notes/installfest.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Hi! The purpose of these notes is to help you install and configure some software that we'll use to, well, make other software. ^_^ Here's a short list of what we'll be installing. Detailed instructions for each item follow. It is really, really important to install everything in this list. It may feel a little tedious, but getting it done now will set you up for success on day 1 (and day 100!) of class.
+
+## Short list of required software:
+
+- [OS X _El Capitan_ (or OS X _Yosemite_)](http://www.apple.com/osx/whats-new/)
+- [Xcode Command Line Tools](#xcode-command-line-tools)
+- [Google Chrome (web browser)](https://www.google.com/chrome/browser/desktop/index.html)
+- [Atom (text editor)](https://atom.io/)
+- [Ruby Version Manager (rvm)](https://rvm.io/)
```
Comment (sudocrystal) [inline lessons/06-environment-setup/notes/installfest.md]: Should this say (package manager) as well?

**Comment by sudocrystal** [comment]
Looks good! Merge it in!

File: lessons/06-environment-setup/README.md (hunk 1 above)
Comment (jnf) [inline lessons/06-environment-setup/README.md]: :+1:

File: lessons/06-environment-setup/notes/installfest.md (hunk 2 above)
Comment (jnf) [inline lessons/06-environment-setup/notes/installfest.md]: Yeah, probably.

**Pull request merged**